countries/tests/test_models.py

```diff
@@ -18,5 +20,10 @@ def test_fields(self):
 
     def test_str(self):
         name = 'Japan'
-        user = CountryFactory.build(name=name)
-        self.assertEqual(str(user), name)
+        country = CountryFactory.build(name=name)
+        self.assertEqual(str(country), name)
```

Comment:
At that point the two tests are the same, right?  So you can just remove this one.